js/apps/account-ui/test/applications.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { expect, test } from "@playwright/test";
+import { login } from "./login";
+
+test.describe("Applications test", () => {
+  test.beforeEach(async ({ page }) => {
+    // Sign out all devices before each test
+    await login(page, "admin", "admin", "master");
+    await page.getByTestId("accountSecurity").click();
+    await page.getByTestId("account-security/device-activity").click();
+
+    await page
+      .getByRole("button", { name: "Sign out all devices", exact: true })
+      .click();
+    await page.getByRole("button", { name: "Confirm" }).click();
+
+    await expect(
+      page.getByRole("heading", { name: "Sign in to your account" }),
+    ).toBeVisible();
+  });
+
+  test("Single application", async ({ page }) => {
+    await login(page, "admin", "admin", "master");
+
+    await page.getByTestId("applications").click();
+
+    await expect(page.getByTestId("applications-list-item")).toHaveCount(1);
+    await expect(page.getByTestId("applications-list-item")).toContainText(
+      process.env.CI ? "Account Console" : "security-admin-console-v2",
```

Comment:
We should probably streamline this at some point so the results are the same between dev and CI.